### PR TITLE
A little refactor 

### PR DESCRIPTION
**Pull request by Lutacon**
- Add missing tags to phpdocs.
- Add some fully-qualifed function calls.
- Add return types.
- Optimize some code.
- Add access modifiers of some constants.

### CODE REVIEW

File: src/Application/Query/Collection.php [hunk 1]
```diff
@@ -29,6 +29,11 @@ class Collection
     public $data;
 
     /**
+     * @param int   $page
+     * @param int   $limit
+     * @param int   $total
+     * @param array $data
```
Comment (jorge07) [inline src/Application/Query/Collection.php]: Unedded

File: src/Infrastructure/User/Auth/Guard/LoginAuthenticator.php [hunk 2]
```diff
@@ -23,9 +23,9 @@
 
 final class LoginAuthenticator extends AbstractFormLoginAuthenticator
 {
-    const LOGIN = 'login';
+    public const LOGIN = 'login';
 
-    const SUCCESS_REDIRECT = 'profile';
+    public const SUCCESS_REDIRECT = 'profile';
```
Comment (jorge07) [inline src/Infrastructure/User/Auth/Guard/LoginAuthenticator.php]: Private?

File: src/Application/Query/Collection.php (hunk 1 above)
Comment (jorge07) [reply src/Application/Query/Collection.php]: If you're using PHP-Cs-fixer, I think the "no_superfluous_phpdoc_tags" is needed. All of this params don't need to be documented.

File: src/Application/Command/User/ChangeEmail/ChangeEmailCommand.php [hunk 3]
```diff
@@ -15,6 +15,11 @@ class ChangeEmailCommand
     /** @var Email */
     public $email;
 
+    /**
+     * ChangeEmailCommand constructor.
```
Comment (jorge07) [inline src/Application/Command/User/ChangeEmail/ChangeEmailCommand.php]: Useless description

File: src/Application/Command/User/SignIn/SignInCommand.php [hunk 4]
```diff
@@ -18,6 +18,11 @@ class SignInCommand
      */
     public $plainPassword;
 
+    /**
+     * SignInCommand constructor.
```
Comment (jorge07) [inline src/Application/Command/User/SignIn/SignInCommand.php]: same

File: src/Domain/User/ValueObject/Email.php [hunk 5]
```diff
@@ -8,6 +8,11 @@
 
 class Email
 {
+    /**
+     * @throws \Assert\AssertionFailedException
+     *
+     * @return Email
```
Comment (jorge07) [inline src/Domain/User/ValueObject/Email.php]: useless return doc

File: src/Infrastructure/User/Query/Projections/UserView.php [hunk 6]
```diff
@@ -20,11 +20,21 @@ class UserView implements UserViewInterface
     /** @var Credentials */
     public $credentials;
 
+    /**
+     * @throws \Assert\AssertionFailedException
+     *
+     * @return UserView
```
Comment (jorge07) [inline src/Infrastructure/User/Query/Projections/UserView.php]: same

File: src/Kernel.php [hunk 7]
```diff
@@ -14,7 +14,7 @@ class Kernel extends BaseKernel
 {
     use MicroKernelTrait;
 
-    const CONFIG_EXTS = '.{php,xml,yaml,yml}';
+    public const CONFIG_EXTS = '.{php,xml,yaml,yml}';
```
Comment (jorge07) [inline src/Kernel.php]: private

**Comment by jorge07** [comment]
@Lutacon add the PGP-CS-fixed to the pipeline can be an options for all of this stuff but we'll need some custom rules

**Comment by Lutacon** [comment]
@jorge07 Yes I think that would be better. I'm gonna add it later

**Comment by jorge07** [comment]
Keep it here, so we've the tool that fixes the style in the same PR

**Comment by jorge07** [comment]
Needs rebase ;)

**Comment by Lutacon** [comment]
@jorge07 take a final look. I just added the CS Fixer to the pipeline. We could do some improvements on CS-Fixer but for now, it's fine.

**Comment by jorge07** [comment]
Awesome @Lutacon

**Pull request merged**